source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -850,33 +850,30 @@ are parsed from the ismaster response in the obvious way.
 Server Description Equality
 ```````````````````````````
 
-For the purposes of updating topology description and publishing SDAM events,
-two server descriptions having the same address
-MUST be considered equal if and only if the values of
-`ServerDescription`_ fields marked (=) are respectively equal.
+For the purpose of determining whether to publish SDAM events, two server
+descriptions having the same address MUST be considered equal if and only if
+the values of `ServerDescription`_ fields marked (=) are respectively equal.
+Two topology descriptions MUST be considered equal if and only if they have
+the same set of server addresses, and the server descriptions for each address
+are respectively equal as just defined.
 
 This specification does not prescribe how to compare server descriptions
 with different addresses for equality.
 
-Note: Server description for each server MUST be updated (replaced)
-every heartbeat. However, new description MUST NOT cause the SDAM flow
-to be executed if the new description is equal, as defined in this section,
-to the previous description.
-
 Updating the TopologyDescription
 ''''''''''''''''''''''''''''''''
 
 Each time the client checks a server,
 it processes the outcome (successful or not)
 to create a `ServerDescription`_,
 and then it processes the ServerDescription to update its `TopologyDescription`_.
+The ServerDescription MUST be processed even if it is equal (as defined in
```

Comment:
nit: I slightly prefer directly linking to sections, rather than referring to them in the paranthetical. I think it makes it much easier to click through (or hover over) while reading:
```
The ServerDescription MUST be processed even if it is considered [equal](/link/to/equality/description)...
```

Reply:
This change is no longer part of this PR.